driver-core/pom.xml

```diff
@@ -224,7 +224,7 @@
                                     JNR does not provide OSGi bundles, so exclude it; the driver can live without it
                                     Explicitly import javax.security.cert because it's required by Netty, but Netty has been explicitly excluded
                                     -->
-                                    <![CDATA[com.google.common.*;version="[16.0.1,21)",!jnr.*,!io.netty.*,javax.security.cert,*]]></Import-Package>
+                                    <![CDATA[com.google.common.*;version="[16.0.1,22)",!jnr.*,!io.netty.*,javax.security.cert,*]]></Import-Package>
```

Comment:
I see that sometimes we use `com.google.common.*` and sometimes `com.google.common*`. Both seem to work but for consistency I'd suggest using one or the other. (Disclaimer: I might be the culprit here.)